### DISCUSSION TITLE
Migrating to Axis.

### DISCUSSION THREAD

**Discussion by Fetakungen** [AxisCommunications/discussions#8] [General] [answered]
Hi, don't really feel that this is the right place to ask this but, we are looking to start using Axis Camera station(coming from milestone) and Cameras but we have quite a few cameras of other brand e.g hikvision. Which connect fine but for exemaple there seems to be no support / plugins for dewarping the image of the hikvision ? Is there a possiblity to dewarp these 3rd party or do this only work with axsis cameras ?

BR,
Anton.

**Comment by mattias-kindborg-at-work** [comment]
I think AXIS Camera Station is built to support both Axis network devices and ONVIF devices. Hikvision would fall into the ONVIF category, and ought to be supported as well as any other ONVIF device. However that sometimes mean that they aren't as tightly integrated as Axis devices.

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
I was wrong! 

It seems that there are custom dewarp settings in AXIS Camera Station. Please have a look at [their documentation](https://help.axis.com/axis-camera-station-5#image-configuration) and search for *Custom dewarp settings*.
